### PR TITLE
pitr: compatibility between pitr and accelerated indexing

### PR DESCRIPTION
**Pull request by Leavrth**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Currently PITR is compatibility with accelerated indexing. So update the document.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):
https://github.com/pingcap/tidb/pull/41670
### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@Leavrth Is this v7.0 related? Could you invite a technical review?

**Review by YuJuncen — APPROVED** [review]

File: faq/backup-and-restore-faq.md [hunk 1]
```diff
@@ -47,11 +47,9 @@ TiKV 支持[动态配置](/tikv-control.md#动态修改-tikv-的配置)自动调
 
 Issue 链接：[#38045](https://github.com/pingcap/tidb/issues/38045)
 
-当前[索引加速功能](/system-variables.md#tidb_ddl_enable_fast_reorg-从-v630-版本开始引入)与 PITR 功能不兼容。在使用索引加速功能时，需要确保后台没有启动 PITR 备份任务，否则可能会出现非预期结果。非预期场景包括：
+当前通过[索引加速功能](/system-variables.md#tidb_ddl_enable_fast_reorg-从-v630-版本开始引入)产生的索引数据无法被 PITR 备份任务备份。
 
-- 如果先启动 PITR 备份任务，再添加索引，此时即使索引加速功能打开，也不会使用加速索引功能，但不影响索引兼容性。
-- 如果先启动添加索引加速任务，再创建 PITR 备份任务，此时 PITR 备份任务会报错，但不影响正在添加索引的任务。
-- 如果同时启动 PITR 备份任务和添加索引加速任务，可能会由于两个任务无法察觉到对方而导致 PITR 不能成功备份增加的索引数据。
+因此，在 PITR 恢复完成后，br 会将通过索引加速功能创建的索引数据删除再重新创建。如果在日志备份期间通过索引加速功能创建的索引很多或索引数据很大，建议在创建索引后进行一次全量备份。
```
Comment (ran-huang) [inline faq/backup-and-restore-faq.md:52]:
```suggestion
因此，在 PITR 恢复完成后，BR 会将通过索引加速功能创建的索引数据删除，再重新创建。如果在日志备份期间通过索引加速功能创建的索引很多或索引数据很大，建议在创建索引后进行一次全量备份。
```

File: system-variables.md [hunk 2]
```diff
@@ -977,12 +977,6 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 > **警告：**
 >
 > 当前索引加速功能未完全兼容添加唯一索引操作。在添加唯一索引时，建议关闭索引加速功能（将 `tidb_ddl_enable_fast_reorg` 设置为 `OFF`）。
->
```
Comment (ran-huang) [inline system-variables.md:980]: 这里建议还是保留一句话，提示有可能需要在 PITR 恢复后进行额外操作。详细内容链接到 faq/backup-and-restore-faq.md

File: faq/backup-and-restore-faq.md (hunk 1 above)
Comment (Leavrth) [reply faq/backup-and-restore-faq.md]: done

File: system-variables.md (hunk 2 above)
Comment (Leavrth) [reply system-variables.md:980]: done

File: system-variables.md [hunk 3]
```diff
@@ -977,12 +977,8 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
...
-> 当关闭 [PITR (Point-in-time recovery)](/br/backup-and-restore-overview.md) 时使用索引加速功能，添加索引的速度预期是 v6.1.0 的 10 倍。但是同时开启 PITR 和索引加速时，不会有性能提升。为了优化性能，建议你先停止 PITR 后台备份任务，以索引加速的方式快速添加索引，然后再启动 PITR 备份任务。否则，可能会发生以下行为：
->
-> - 如果先启动 PITR 备份任务，再添加索引，此时即使索引加速功能打开，添加索引任务默认会自动回退到旧模式，即以较慢的速度添加索引，但 PITR 的备份任务能够正常运行并备份新增的索引数据。
-> - 如果先启动添加索引加速任务，再尝试启动 PITR 备份任务，此时 PITR 备份任务会抛出错误并退出，但这不会影响正在添加索引的任务。在索引加速任务完成后，你需要重新启动 PITR 日志备份任务，并手动执行一次全量备份。
-> - 如果同时启动 PITR 备份任务和添加索引加速任务，可能会由于两个任务无法察觉到对方而导致 PITR 不能成功备份新增的索引数据。在添加索引任务完成后，你仍需要重新启动 PITR 日志备份任务，并手动执行一次全量备份。
+> 
+> 目前，PITR 恢复会对日志备份的时间段内通过索引加速功能添加的索引进行额外操作，以达到兼容效果，详细内容请参考 [索引加速功能为什么与 pitr 功能不兼容](faq/backup-and-restore-faq.md#索引加速功能为什么与-pitr-功能不兼容)。
```
Comment (ran-huang) [inline system-variables.md:981]:
```suggestion
> 目前，PITR 恢复会对日志备份的时间段内通过索引加速功能添加的索引进行额外操作，以达到兼容效果，详细内容请参考 [索引加速功能为什么与 PITR 功能不兼容](faq/backup-and-restore-faq.md#索引加速功能为什么与-pitr-功能不兼容)。
```

**Review by ran-huang — APPROVED** [review]
Rest LGTM

File: faq/backup-and-restore-faq.md [hunk 4]
```diff
@@ -47,11 +47,9 @@ TiKV 支持[动态配置](/tikv-control.md#动态修改-tikv-的配置)自动调
 
 Issue 链接：[#38045](https://github.com/pingcap/tidb/issues/38045)
 
-当前[索引加速功能](/system-variables.md#tidb_ddl_enable_fast_reorg-从-v630-版本开始引入)与 PITR 功能不兼容。在使用索引加速功能时，需要确保后台没有启动 PITR 备份任务，否则可能会出现非预期结果。非预期场景包括：
+当前通过[索引加速功能](/system-variables.md#tidb_ddl_enable_fast_reorg-从-v630-版本开始引入)产生的索引数据无法被 PITR 备份任务备份。
```
Comment (Oreoxmt) [inline faq/backup-and-restore-faq.md:50]:
```suggestion
当前通过[索引加速功能](/system-variables.md#tidb_ddl_enable_fast_reorg-从-v630-版本开始引入)创建的索引数据无法被 PITR 备份。
```

File: system-variables.md [hunk 5]
```diff
@@ -977,12 +977,8 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
...
 >
 > 当前索引加速功能未完全兼容添加唯一索引操作。在添加唯一索引时，建议关闭索引加速功能（将 `tidb_ddl_enable_fast_reorg` 设置为 `OFF`）。
->
-> 当关闭 [PITR (Point-in-time recovery)](/br/backup-and-restore-overview.md) 时使用索引加速功能，添加索引的速度预期是 v6.1.0 的 10 倍。但是同时开启 PITR 和索引加速时，不会有性能提升。为了优化性能，建议你先停止 PITR 后台备份任务，以索引加速的方式快速添加索引，然后再启动 PITR 备份任务。否则，可能会发生以下行为：
->
-> - 如果先启动 PITR 备份任务，再添加索引，此时即使索引加速功能打开，添加索引任务默认会自动回退到旧模式，即以较慢的速度添加索引，但 PITR 的备份任务能够正常运行并备份新增的索引数据。
-> - 如果先启动添加索引加速任务，再尝试启动 PITR 备份任务，此时 PITR 备份任务会抛出错误并退出，但这不会影响正在添加索引的任务。在索引加速任务完成后，你需要重新启动 PITR 日志备份任务，并手动执行一次全量备份。
-> - 如果同时启动 PITR 备份任务和添加索引加速任务，可能会由于两个任务无法察觉到对方而导致 PITR 不能成功备份新增的索引数据。在添加索引任务完成后，你仍需要重新启动 PITR 日志备份任务，并手动执行一次全量备份。
+> 
+> 目前，PITR 恢复会对日志备份的时间段内通过索引加速功能添加的索引进行额外操作，以达到兼容效果，详细内容请参考 [索引加速功能为什么与 PITR 功能不兼容](faq/backup-and-restore-faq.md#索引加速功能为什么与-pitr-功能不兼容)。
```
Comment (Oreoxmt) [inline system-variables.md:981]:
```suggestion
> 目前，PITR 恢复会额外处理日志备份时间段内通过索引加速功能创建的索引，以达到兼容效果。详细内容请参考[索引加速功能为什么与 PITR 功能不兼容](faq/backup-and-restore-faq.md#索引加速功能为什么与-pitr-功能不兼容)。
```

**Review by Oreoxmt — COMMENTED** [review]
Rest LGTM

**Review by Oreoxmt — APPROVED** [review]

File: system-variables.md [hunk 6]
```diff
@@ -977,12 +977,8 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 > **警告：**
 >
 > 当前索引加速功能未完全兼容添加唯一索引操作。在添加唯一索引时，建议关闭索引加速功能（将 `tidb_ddl_enable_fast_reorg` 设置为 `OFF`）。
```
Comment (tangenta) [inline system-variables.md:979]:
```suggestion
```

**Pull request merged**